src/resolution.js

```diff
@@ -25,6 +32,8 @@ var getScreenTypeObject = function (type) {
 * @module enyo/resolution
 */
 var ri = module.exports = {
+	config: {},
```

Comment:
I guess not. But I figured this may need to be present if the RI object was referenced separately from the initialization.